src/d/actor/d_a_npc_photo.cpp

```diff
@@ -4,337 +4,2266 @@
+    0x2A74,
+    0x2A75,
+    0x0000,
+};
+
+static u32 l_msg_talk_photo_3[] = {
+    0x2A77,
+    0x2A78,
+    0x0000,
+};
+
+static u32 l_msg_talk_photo_4[] = {
+    0x2A7A,
+    0x2A7B,
+    0x0000,
+};
+
+static u32 l_msg_talk_photo_5[] = {
+    0x2A7D,
+    0x2A7E,
+    0x0000,
+};
+
+static u32 l_msg_talk_photo_6[] = {
+    0x2A80,
+    0x2A81,
+    0x0000,
+};
+
+static u32* l_msg_talk_photo[] = {
+    l_msg_talk_photo_0, l_msg_talk_photo_1, l_msg_talk_photo_2, l_msg_talk_photo_3,
+    l_msg_talk_photo_4, l_msg_talk_photo_5, l_msg_talk_photo_6,
+};
+
+static u32 l_msg_buy_photo[] = {
+    0x3786,
+    0x0000
+};
+
+static u32 l_msg_get_photo[] = {
+    0x3785,
+    0x0000
+};
+
+static u32 l_msg_2F[] = {
+    0x2A82,
+    0x2A83,
+    0x2A84
+};
+
+static u32 l_msg_down[] = {
+    0x2A86,
+    0x0000
+};
+
+static u32 l_msg_ub4[] = {
+    0x2D62,
+    0x0000,
+};
+
+static u32 l_msg_1st_order_xy[] = {
+    0x2A5E,
+    0x2A5F,
+    0x2A60,
+    0x0002
+};
+
+static u32 l_msg_2nd_order_xy[] = {
+    0x2A61,
+    0x2A62,
+    0x2A60,
+    0x0004
+};
+
+static u32 l_msg_3rd_order_xy[] = {
+    0x2A64,
+    0x2A60,
+    0x0006
+};
+
+static u32 l_msg_color_xy[] = {
+    0x2A58,
+    0x2A59,
+    0x2A5A,
+    0x0007,
+    0x2A85,
+    0x0008,
+    0x2A5B,
+    0x0000
+};
+
+static u32 l_msg_xy_buy_photo[] = {
+    0x3788,
+    0x3789,
+    0x0009,
+    0x378C,
+    0x000A,
+    0x378D,
+    0x000B,
+    0x3786,
+    0x0000
+};
+
+static dCcD_SrcCyl l_cyl_src2 = {
+    // dCcD_SrcGObjInf
+    {
+        /* Flags             */ 0,
+        /* SrcObjAt  Type    */ 0,
+        /* SrcObjAt  Atp     */ 0,
+        /* SrcObjAt  SPrm    */ 0,
+        /* SrcObjTg  Type    */ 0,
+        /* SrcObjTg  SPrm    */ 0,
+        /* SrcObjCo  SPrm    */ cCcD_CoSPrm_NoCoHitInfSet_e | cCcD_CoSPrm_VsGrpAll_e | cCcD_CoSPrm_IsOther_e | cCcD_CoSPrm_Set_e,
+        /* SrcGObjAt Se      */ 0,
+        /* SrcGObjAt HitMark */ 0,
+        /* SrcGObjAt Spl     */ 0,
+        /* SrcGObjAt Mtrl    */ 0,
+        /* SrcGObjAt SPrm    */ 0,
+        /* SrcGObjTg Se      */ 0,
+        /* SrcGObjTg HitMark */ 0,
+        /* SrcGObjTg Spl     */ 0,
+        /* SrcGObjTg Mtrl    */ 0,
+        /* SrcGObjTg SPrm    */ dCcG_TgSPrm_NoHitMark_e,
+        /* SrcGObjCo SPrm    */ 0,
+    },
+    // cM3dGCylS
+    {
+        /* Center */ 0.0f, 0.0f, 0.0f,
+        /* Radius */ 40.0f,
+        /* Height */ 160.0f,
+    },
+};
+
+static char* l_npc_staff_id[] = {
+    "Po"
+};
+
+static const int l_bck_ix_tbl[] = {
+    0x00,
+    0x05,
+    0x06,
+    0x07,
+    0x04,
+};
+
+static const int l_btp_ix_tbl[] = {
+    PO_BTP_MABA01,
+    PO_BTP_MABA02
+};
+
+struct SaveDatStruct {
+    /* 0x00 */ u16 field_0x00;
+    /* 0x02 */ u16 field_0x02;
+    /* 0x04 */ u16 field_0x04;
+    /* 0x06 */ u16 field_0x06;
+}; // Size: 0x08
+
+static const SaveDatStruct l_save_dat[] = {
+    {
+        0x1208,
+        0x1701,
+        0x1601,
+        0xC407,
+    }
+};
```

Comment:
```suggestion
static const SaveDatStruct l_save_dat = {
    0x1208,
    0x1701,
    0x1601,
    0xC407,
};
```

Reply:
This one's fine since it has the extra braces already. Even though it's only one element long the devs probably copy pasted these from another actor or a template where it's longer.